src/main/java/nz/cri/gns/NZSHM22/util/DataLogger.java

```diff
@@ -0,0 +1,192 @@
+package nz.cri.gns.NZSHM22.util;
+
+import cern.colt.matrix.tdouble.DoubleMatrix2D;
+import java.io.*;
+import java.nio.file.Files;
+import java.text.DecimalFormat;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+public class DataLogger {
+
+    static DecimalFormat fmt = new DecimalFormat("0.###E0");
```

Comment:
I know this is just copied from it's previous location, but is there nothing in java std lib that does this?

Reply:
`DecimalFormat` is from the standard Java libraries